dataactbroker/fileRoutes.py

```diff
@@ -275,7 +275,11 @@ def generate_file(submission_id, file_type):
     def generate_detached_file(file_type, cgac_code, frec_code, start, end):
         """ Generate a file from external API, independent from a submission """
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
-        return file_manager.generate_detached_file(file_type, cgac_code, frec_code, start, end)
+        if cgac_code or frec_code:
```

Comment:
If you're returning an error you can check `if not cgac_code and not frec_code` and then return. Then you don't need an else. This isn't necessarily incorrect but we want to stay consistent.